expyfun/stimuli/_tracker.py

```diff
@@ -0,0 +1,862 @@
+"""Adaptive tracks for psychophysics (individual, or multiple randomly dealt)
+"""
+# Author: Ross Maddox <ross.maddox@rochester.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import time
+from scipy.stats import binom
+import json
+import matplotlib.pyplot as plt
```

Comment:
When I do ``make html_dev`` locally I still get the same error as CircleCI is getting when trying to build the documentation.